MailCore/Utils/MailUserDefaults+Extension.swift

```diff
@@ -199,6 +201,30 @@ public extension UserDefaults {
         }
     }
 
+    var isMatomoAuthorized: Bool {
```

Comment:
Nice to have, someday it would be nice to have a common read/write generic method that takes the key as argument  (like: `.matomoAuthorized`). That way we would reduce the duplication of code.

It looks like that's code duplication SonarCube is not able to detect.

Reply:
I created a task on Trello so we can discuss it at our next daily